pkg/apis/app/v1alpha1/kogitoservices_types.go

```diff
@@ -157,3 +157,17 @@ func (k *KogitoServiceSpec) GetResources() corev1.ResourceRequirements { return
 func (k *KogitoServiceSpec) SetResources(resources corev1.ResourceRequirements) {
 	k.Resources = resources
```

Comment:
I think we should make a decision about this. Either we check null pointers everywhere, or nowhere or we change the approach about adding functions vs methods.
What do you think?

Reply:
I think we can use this approach:
Preferred way of declaring such methods is to declare them on struct, not pointer.
If method is declared on pointer then it should handle case when pointer is nil.

@ricardozanini @radtriste  What do you think about it?

Reply:
good question. for me it should be checked before the method is called and not when it is called...